.github/workflows/disk-cleanup.yml

```diff
@@ -0,0 +1,16 @@
+name: Disk Cleanup
```

Comment:
This is a workflow, not an action. I think this needs to be an action so it runs inside another workflow.

See this for how to create an action:

https://github.com/stellar/actions/blob/016d5d949aa4dad4448ddb809c95d7ad02229d4c/rust-cache/action.yml#L1-L61

Reply:
ah, never knew about composite actions. That's neat!